packages/orbit-components/src/Stack/__tests__/index.test.tsx

```diff
@@ -23,9 +20,8 @@ describe("Stack", () => {
 
     expect(screen.getByTestId(dataTest)).toBeInTheDocument();
     expect(screen.getByRole("button", { name: "Sign In" })).toBeInTheDocument();
+
     expect(screen.getByTestId(dataTest)).toHaveStyle({
-      gap: "16px",
-      width: "100%",
       flexDirection: "row",
       flexWrap: "nowrap",
       flexShrink: "0",
```

Comment:
Small 💅 I know this wasn't you, and I missed it on first review, but can you please make it compliant with the extensive property (string) we're following 🙏